CHANGELOG.md

```diff
@@ -2,7 +2,32 @@
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
-## [Unreleased](https://github.com/trailofbits/manticore/compare/0.1.4...HEAD)
+## [Unreleased](https://github.com/trailofbits/manticore/compare/0.1.5...HEAD)
+
+## 0.1.5 - 2017-09-15
+
+### Deprecated
+
+- `Manticore('binary', ['arg1', 'arg2'])` style initialization. Use new class methods (see below).
+
+### Added
+
+- Manticore class methods for Manticore initialization.
```

Comment:
That is phrased in a redundant manner. Maybe: `Platform-specific class methods for Manticore initialization`?